tests/functional/ancestral/cram/general.t

```diff
@@ -0,0 +1,39 @@
+Setup
+
+  $ source "$TESTDIR"/_setup.sh
+
+General running of augur ancestral.
+- Note that the (parsimonious) C18T on the branch of Sample_C is inferred by augur
+as a root mutation of C18T + T18C reversion on Node_AB. This is reflected in the
+node-data JSON we diff against.
+- We supply the refererence sequence so mutations are called at the root.
+- Ambiguous nucleotides (there are 3 Ns in sample_C) are inferred (default)
+
+  $ ${AUGUR} ancestral \
+  >  --tree $TESTDIR/../data/simple-genome/tree.nwk \
```

Comment:
_(non-blocking, more about consistency)_

```suggestion
  >  --tree "$TESTDIR"/../data/simple-genome/tree.nwk \
```

This isn't a dealbreaker because `$TESTDIR` is known to not have spaces, but it's used across other tests probably just as good practice. Recommendation applies generally to variables in this test file.

Reply:
Yeah good shout - I think I've fixed these across all the modified tests.

For my understanding, if we have something like `DATA="$TESTDIR/../data"` - do we then need to quote every string which includes `$DATA`? For instance, must `--tree "${DATA}/tb/tree.nwk"` be quoted like so, or is `--tree ${DATA}/tb/tree.nwk` going to be ok? (Assuming of course that `$DATA` has a space in it.)

For instance, across the board we do 
```bash
export AUGUR="${AUGUR:-$TESTDIR/../../../../bin/augur}" # appropriate quoting
$ {AUGUR} translate ... # no quoting
```

Reply:
For the `DATA` example, yes that requires quoting. Here's an example: 4281ed8727fcb1f850ca7b61126eabe713ff4a60

For `$AUGUR`, if I were writing all the tests from scratch, I would add quotes. I think it's easier to always quote as a rule of thumb, even if the value is known to not have spaces, just so I (and readers) don't have to think about whether it might contain spaces now or in the future.

Reply:
Following up for anyone revisiting this issue

We _cannot_ use quoting for `$AUGUR` because our CI does not want it treated as a single word:

https://github.com/nextstrain/augur/blob/9cd915a2a8a2d63244fc7f7d0316b51f5c3ef936/.github/workflows/ci.yaml#L67-L68

Reply:
Oh that's a good point, thanks for noting! Unquoted should be fine as long as we don't use spaces in our test paths.